src/web/app/pages-student/student-profile-page/student-profile-page.component.html

```diff
@@ -1,3 +1,114 @@
-<p>
-  student-profile-page works!
-</p>
+<div *ngIf="student">
+  <h1>Student Profile</h1>
+  <div class="card">
+    <div class="card-body fill-plain">
+      <h1>{{editForm.get('studentshortname').value}}</h1>
```

Comment:
This part of the form displays the student's name as registered by the instructor. It is not the short name.